test/lib/validator.test.js

```
@@ -136,3 +136,121 @@ describe('assert',  () => {
     })
   })
 })
+
+
+describe('assertMany',  () => {
+
+  describe('when not providing the validation functions as an array', () => {
+    test('should throw an Error', () => {
+
+      const resourceId = uuidv4()
+      const resource = undefined
+
+      expect(() => {
+        assertMany(() => canFind(resource, `Resource ${resourceId}`))
+      }).toThrow()
+    })
+  })
+
+  describe('when used with a single validation function', () => {
+    test('should return a correct response when validation fails', () => {
+
+      const resourceId = uuidv4()
+      const resource = undefined
+
+      try {
+        assertMany([ () => canFind(resource, `Resource ${resourceId}`) ])
+      } catch (e) {
+        const expected = `Can\'t find: Resource ${resourceId}`
+        expect(e.message).toEqual(expected)
+      }
+    })
+
+    test('should return empty array for a passed validation', () => {
+
+      const resourceId = uuidv4()
+      const resource = { id: resourceId }
+
+      const errorMessages = assertMany([ () => canFind(resource, `Resource ${resourceId}`) ])
+      const expected = []
+
+      expect(errorMessages).toEqual(expect.arrayContaining(expected))
+    })
+  })
+
+  describe('when used with multiple validation functions', () => {
+    test('should return a correct response when both validation fail', () => {
```

I believe assertMany will fail fast on the first failed validation
(to collect all failures was the role of the deprecated `validateMany`, iirc)